### PR TITLE
Disable sticky headers for menus and courses

### LINKED ISSUE #3348: List items are misbehaving
I think something is mutating? When I scroll the menu list, it sort of... jitters. After that strange UI animation, it inserts a whole bunch of duplicate items in the list each time it loads. It continues to do so when I continue to scroll. Could you help debug this, please, @hannesmcman?

### PR DESCRIPTION
**Pull request by drewvolz**
Closes #3348

If we can compromise on the headers not being sticky for `FancyMenu` and `CourseSearch`, this appears to hide the react native issue with headers not remembering their positions when a list is scrolling.

I do not think we need to apply this to any of the other `SectionList` components.

Thoughts?

### CODE REVIEW

**Review by hawkrives — DISMISSED** [review]
Before we merge this, I want someone to tell me when this was introduced. Because this wasn't an issue on Saturday's build, so somehow something we've done in the last three days broke this.

And I want to know what, instead of just papering over it.

There haven't been any giant branches merged in that timeframe, so you should just be able to bisect everything.

**Comment by hawkrives** [comment]
For instance, I don't seem to be able to reproduce this problem on build 43564, from two days ago, although I may just not be doing it right.

**Comment by hawkrives** [comment]
Ah, I missed that the bug only occurs when there are no filters applied, so we only just exposed it.

File: modules/food-menu/fancy-menu.js
Comment (hawkrives) [inline modules/food-menu/fancy-menu.js:211]: Please include the relevant RN issue numbers in these comments

**Review by hawkrives — CHANGES_REQUESTED** [review]

**Comment by hawkrives** [comment]
@drewvolz Can you remind me why Calendar and Contacts and Streams can keep using the sticky headers? What makes them different?

**Comment by drewvolz** [comment]
> Ah, I missed that the bug only occurs when there are no filters applied, so we only just exposed it. 

Exactly. The broken filters have hidden this problem from us. It’s only just now that it’s visible.

> Can you remind me why Calendar and Contacts and Streams can keep using the sticky headers? What makes them different?

I haven’t been able to reproduce this issue on them. I was only trying to tweak the noticeably impacted lists. It seems to me that our “big networked lists” (that can load a ton of rows that change via the filters) are the impacted ones.

If you’d prefer for it to be applied to all SectionLists, that’s fine, too.

**Comment by hawkrives** [comment]
No, I'd just like to understand why it only seems to be the filtered lists. Streams and Hours can get pretty large… (I understand that you don't have the answers either.)

File: modules/food-menu/fancy-menu.js
Comment (rye) [reply modules/food-menu/fancy-menu.js:211]: I second this—can you include `see facebook/react-native#20956, facebook/react-native#21468, facebook/react-native#22444, facebook/react-native#20127` here?

File: source/views/sis/course-search/list.js
Comment (rye) [inline source/views/sis/course-search/list.js:124]: Same here.

**Pull request closed without merging**
